docs/documentation.md

```diff
@@ -1,23 +1,26 @@
 # How To Document Components
 
-We rely on inline documentation and a markdown example file for both [the docs site on github.io,](https://woocommerce.github.io/woocommerce-admin/#/) and the DevDocs section in wp-admin. This allows us to keep the docs site up-to-date if any components change, because the documentation is kept in the same place as the code. Read on for how to build out the docs files and devdocs section.
+We rely on a markdown documentation file for [the docs site on github.io,](https://woocommerce.github.io/woocommerce-admin/#/) and an example JavaScript file for the DevDocs section in wp-admin. Read on for how to build out the docs files and devdocs section.
 
-## 1. Add the inline documentation to generate a markdown file
+## 1. Create a markdown documentation file
```

Comment:
Might be worth noting that the base `README.md` file can be autogenerated and description and props can be automatically pulled from the class comment and proptypes.

Since this is triggered via `npm run docs` maybe we can move this step beneath step 3.

Reply:
Updated this in e0f3f3380df6a25a9a019d9b9ec081306fece036